effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -385,7 +349,7 @@ object Transformer {
         emit(callLabel(returnAddress, values.map(transform)))
         RetVoid()
 
-      case machine.NewStack(variable, prompt, frame, rest) =>
+      case machine.NewStack(variable, prompt, frame, isRegion, rest) =>
```

Comment:
will this flag go away when we allocate regions directly on the stack?

Reply:
No, we need to know whether we are returning from a region.